circle.yml

```diff
@@ -0,0 +1,62 @@
+# Setup in CircleCI account the following ENV variables:
+# BINTRAY_ORGANIZATION
+# BINTRAY_ACCOUNT
+# BINTRAY_API_KEY
+general:
+  # Don't run CI for PR, only for major branches
+  branches:
+    only:
+      - master
+      - /v[0-9]+(\.[0-9]+)*/
+      - feature/circleci
```

Comment:
Same as previous note.

I think this limitation on build could be removed, so we run tests on every PR, since it's really fast for `st2web`.

I assume this `circle.yml` file is from `st2` branch (now it's still expensive to run build on every PR).
For `st2-packages` we run builds on every PR: https://github.com/StackStorm/st2-packages/blob/master/circle.yml#L8